docs/guides/operator/customizing-keycloak.adoc

```diff
@@ -43,10 +43,11 @@ spec:
     hostname: test.keycloak.org
 ----
 
-[NOTE]
-====
-With custom images, every build time option passed either through a dedicated field or the `additionalOptions` is ignored.
-====
+NOTE: With custom images, every build time option passed either through a dedicated field or the `additionalOptions` is ignored.
+
+NOTE: Operator is unaware of any TLS settings in your `Dockerfile` when using a custom image.
+It may cause the management interface uses the `https` schema, but the Operator accesses it via `http`.
+To ensure proper TLS configuration, use the `tlsSecret` and `truststores` properties in the Keycloak CR.
```

Comment:
I'd maybe make it more generic. Operator is unaware of any config options baked in a custom image. This is problematic namely TLS and HTTP(S) ports settings which Operator uses when configuring Services.
WDYT?